src/v1/app.ts

```diff
@@ -207,15 +207,19 @@ class AppController {
 
     onHexViewerSelectionChanged() {
         //console.log("setSelection", ui.hexViewer.selectionStart, ui.hexViewer.selectionEnd);
-        localStorage.setItem("selection", JSON.stringify({ start: this.ui.hexViewer.selectionStart, end: this.ui.hexViewer.selectionEnd }));
+        localStorage.setItem("selection", JSON.stringify({
+            start: this.ui.hexViewer.selectionStart,
+            end: this.ui.hexViewer.selectionEnd,
+            cursor: this.ui.hexViewer.selectionCursor
+        }));
 
         var start = this.ui.hexViewer.selectionStart;
         var hasSelection = start !== -1;
 
         this.refreshSelectionInput();
 
         if (this.ui.parsedDataTreeHandler && hasSelection && !this.selectedInTree) {
-            var intervals = this.ui.parsedDataTreeHandler.intervalHandler.searchRange(this.ui.hexViewer.mouseDownOffset || start);
+            var intervals = this.ui.parsedDataTreeHandler.intervalHandler.searchRange(this.ui.hexViewer.selectionCursor || start);
```

Comment:
Makes perfect sense, thanks! 👍

Currently one has to click on some byte using mouse in order to highlight the relevant field in the _object tree_, so indeed, the keyboard control of the Web IDE is quite limited.

To be clear, I'm still convinced that `selectionCursor` variable is unnecessary and something like `this.ui.hexViewer.isSelectionCursorAtEnd ? this.ui.hexViewer.selectionEnd : this.ui.hexViewer.selectionStart` would do better. I just like the idea.